.flake8

```diff
@@ -0,0 +1,41 @@
+# Can't yet be moved to the pyproject.toml due to https://gitlab.com/pycqa/flake8/-/issues/428#note_251982786
```

Comment:
@asottile moved the project to GitHub, losing all the metadata and breaking all the links, so idk which comment this linked to. The migrated issue is this one: https://github.com/PyCQA/flake8/issues/234

Reply:
Updated link